nucypher/blockchain/eth/sol/source/contracts/MiningAdjudicator.sol

```diff
@@ -54,7 +54,7 @@ contract MiningAdjudicator is Upgradeable {
         public
     {
         // Sanity checks.
-        require(address(_escrow) != address(0) &&  // This contract has an escrow, and it's not the null address.
+        require(_escrow.secondsPerPeriod() > 0 &&  // This contract has an escrow, and it's not the null address.
```

Comment:
:+1:

Reply:
Yeah - this is a gain for readability :+1: